### ISSUE TITLE
Update to NES.css 1.0.0

### ISSUE THREAD

**Issue by guastallaigor** [bschulte/nes-react#1]
Hello! We've just released version 1.0.0 on NES.css, and launch a few others components/icons/classes, and almost every class is going to break because we added a `nes-` prefix to every class component. So it would be nice to have this wrapper updated aswell :smile: 

If I can help you in anyway just let me know :+1: 

Thank you.

**Comment by bschulte** [comment]
Absolutely! I've been wanting to work a bit more on this so now's the perfect time :)

I'll let you know if I have any issues/questions.
